packages/strapi-generate/lib/target.js

```diff
@@ -56,16 +56,27 @@ function generateTarget(options, cb) {
         return sb(err);
```

Comment:
did you cahnge something in this file or just prettier ?

Reply:
Just prettier